doc/source/whatsnew/v1.0.0.rst

```diff
@@ -217,6 +217,7 @@ Categorical
 
 - Added test to assert the :func:`fillna` raises the correct ValueError message when the value isn't a value from categories (:issue:`13628`)
 - Bug in :meth:`Categorical.astype` where ``NaN`` values were handled incorrectly when casting to int (:issue:`28406`)
+- Bug in :meth:`Categorical.astype` not allowing for casting to extension dtypes (:issue:`28668`)
```

Comment:
can you add *another* note about the merge issue (which was the OP); IIRC its the same issue number. otherwise lgtm, ping on green.

Reply:
@jreback Included a note for the merge issue, let me know if it looks good and thanks for the review